src/pages/admin/team/components/translate-member-form/TranslateMemberForm.tsx

```diff
@@ -0,0 +1,109 @@
+import { forwardRef, useMemo } from 'react';
+import { TEAM_MEMBER_VALIDATION_FUNCTIONS } from '@/validation/admin/team-member-schema/team-member-schema';
+import { useFormManager } from '@/hooks/admin/use-form-manager/useFormManager';
+import { Select } from '@/components/common/select/Select';
+import { Button } from '@/components/admin/button/Button';
+import { COMMON_TEXT_ADMIN, LANGUAGES } from '@/const/admin/common';
+import { CommonMemberFields } from '../common-member-fields/CommonMemberFields';
+import { VisibilityStatus } from '@/types/admin/common';
+import styles from './TranslateMemberForm.module.scss';
+import './TranslateMemberForm.scss';
+import { useCommonMemberFields } from '@/hooks/admin/use-common-member-fields/useCommonMemberFields';
+
+export interface TranslateTeamMemberFormValues {
+    fullName: string;
+    description: string;
+}
+export interface TranslateTeamMemberFormErrorState {
+    fullName?: string;
+    description?: string;
+    [key: string]: string | undefined;
+}
```

Comment:
I believe it can be changed to this, try this
```suggestion
export type TranslateTeamMemberFormErrorState = Partial<Record<keyof TranslateTeamMemberFormValues, string>>;
```